### ISSUE TITLE
multi column pdf file text extraction

### ISSUE THREAD

**Issue by sanketpatel91** [pymupdf/RAG#78]
Hello,
I am reaching out regarding my recent experience with pymupdf4llm. I have a PDF file that was created from a PowerPoint presentation, and I am attempting to extract specific text elements from it.

pdf content :
Text 1
          - sub text 1.1
          - sub text 1.2

Text 2
        - sub text 2.1
        - sub text 2.2
        
I am currently using the following code to read the PDF file:

```
all_pages_pdf = pymupdf4llm.to_markdown(filename, `page_chunks=True)
    for page in all_pages_pdf:
		page_number = page['metadata']['page']
		page_content = page['text']
		print(page_number)
		print(page_content)
```

Actual Output With V0.0.10 code :
Text 1
Text 2

- sub text 1.1
- sub text 1.2

- sub text 2.1
- sub text 2.2

However, I am aiming for the following desired output:
Text 1
- sub text 1.1
- sub text 1.2

Text 2
- sub text 2.1
- sub text 2.2

I would appreciate any guidance or assistance in achieving the desired output.
Thank you for your attention to this matter.

**Comment by JorjMcKie** [comment]
Without an example file there is no way to deal with this issue.

**Comment by ITHealer** [comment]
> Without an example file there is no way to deal with this issue.

[paper02.pdf](https://github.com/user-attachments/files/16435820/paper02.pdf)

![image](https://github.com/user-attachments/assets/756ab6be-0c62-4392-b650-15974254b60e)

When I use "pymupdf4llm" to extract information, it is not working correctly for papers. 
Do you have any idea to fix this problem?

Thanks!

**Comment by ITHealer** [comment]
@JorjMcKie

**Comment by afonsoguerra** [comment]
I'm also seeing issues with multi-column PDFs, just try to get anything from Science Magazine (for copyright reasons I can't paste any here". Works great for simple 1 column PDFs though, so thanks for that.

**Comment by JorjMcKie** [comment]
@ITHealer Why do you append your issue to another one that has nothing to do with your topic?

Please open a separate issue so I can deal appropriately with the original post.

**Comment by JorjMcKie** [comment]
Fixed in v0.0.17.
